tests/unit/configuration_subsystem/test_posix_message_queue.py

```diff
@@ -0,0 +1,36 @@
+"""Tests related to a missing ``/dev/mqueue`` directory when using ``podman``."""
+from pathlib import Path
+
+
+def exists(_arg):
+    """Return false when checking for a file or directory path.
+
+    :param _arg: The file or directory path to look for
+    :returns: False, indicating it does not exist
+    """
+    return False
+
+
+def test_posix_message_queue_ee(monkeypatch, generate_config):
+    """Check for error messages related to missing ``/dev/mqueue`` and ``podman`` when ee true.
+
+    :param monkeypatch: The monkeypatch fixture
+    :param generate_config: The configuration generator fixture
+    """
+    monkeypatch.setattr(Path, "exists", exists)
```

Comment:
Since this is copy-pasted in all the tests, I recommend putting it into a local fixture.

Reply:
tests were completely rewritten....